contracts/interfaces/IERC1967.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @dev ERC-1967: Proxy Storage Slots. This interface contains the events defined in the ERC.
+ *
+ * _Available since v4.9._
```

Comment:
This is not correct since it's released as part of `v4.8.3`. Will open a PR @Amxx.

Reply:
FYI https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4183